recipe/meta.yaml

```diff
@@ -1,8 +1,8 @@
-{% set conda_version = "24.9.2" %}
+{% set conda_version = "24.11.0" %}
```

Comment:
Are we intentionally wanting this version or should we be using the latest `24.11.2`?